src/main/scala/org/apache/mesos/chronos/scheduler/jobs/TaskUtils.scala

```diff
@@ -32,6 +32,15 @@ object TaskUtils {
     taskIdPattern.findFirstIn(taskIdString).nonEmpty
   }
 
```

Comment:
I would have preferred to use Scalaz's method on Booleans (see option[A](a: => A) in http://docs.typelevel.org/api/scalaz/nightly/index.html#scalaz.syntax.std.BooleanOps)

and encourage more functional style. I am pretty sure there are more cases where it can be used, but if you prefer the more verbose style of
if (condition)
   Some(value)
else
   None

It is up to you.